app/scripts/vendor/env-test.js

```diff
@@ -81,4 +81,11 @@
   if (isStyleAllowed(style)) {
     docElement.className += (' ' + style);
   }
+
+  /**
+   * Check if the user is signing in to Sync on Firefox for iOS. User agent sniff. Yuck.
+   */
+  if (/FxiOS/.test(navigator.userAgent) && parseQueryParams().service === 'sync') {
```

Comment:
Other notable place that we use [`userAgent`](https://github.com/mozilla/fxa-content-server/search?utf8=%E2%9C%93&q=useragent):
- [/app/scripts/views/marketing_snippet.js:66](https://github.com/mozilla/fxa-content-server/blob/0a02b12caf9a69d021963a5b0fb68afa1d687fa9/app/scripts/views/marketing_snippet.js#L66)

    ```js
      var ua = this.window.navigator.userAgent;

      // covers both B2G and Firefox for Android
      var isMobileFirefox = /Mobile/.test(ua) && /Firefox/.test(ua);
      var isTabletFirefox = /Tablet/.test(ua) && /Firefox/.test(ua);
```